src/porepy/applications/test_utils/models.py

```diff
@@ -114,24 +112,18 @@ class Model(geometry, model_class):
         pass
 
     # Create an instance of the combined class
-    model = Model(params)
+    model: pp.PorePyModel = cast(pp.PorePyModel, Model(params))
```

Comment:
Is it necessary to both cast and declare?

Reply:
No, removing declaration.